Serializer/PlainTextDeserializationVisitor.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Smartbox\CoreBundle\Serializer;
+
+/**
+ * This is a really simple visitor that will effectively bypass JMS. Useful for REST APIs that are not truly REST APIs
+ * and return plain text responses, which JMS by default refuses to process because they are not real JSON.
```

Comment:
Instead of having the comment of the class saying this, I would rather create an architecture decision record with that information, because this is how we are going to use to, but other people could use for different purposes

Reply:
I added a simple ADR but I'd keep the comment here anyway. Class descriptions are an immediate source of information. ADR records would be the last place I'd look for information when trying to figure out what a class does.

Reply:
Class description are there to explain the purpose of the class, and on the part of 
>  which JMS by default refuses to process because they are not real JSON.

I think is going far from the goal of this class. This is describing how in another bundle we implement this, but this bundle is open source and could be used totally independent

My suggestion is to add the comments taking into account that core-bundle is a standalone bundle. Maybe we could leave this comment to something like this
```
 * This is a really simple visitor that will effectively bypass JMS. Useful for cases when JMS receives a plain text to deserialize
```

Reply:
Done